ui/app/src/components/PluginDetailsView/PluginDetailsView.tsx

```diff
@@ -315,8 +338,16 @@ export default function PluginDetailsView(props: PluginDetailsViewProps) {
                 />
               </Alert>
             )}
-
             <Typography variant="body1">{description}</Typography>
+            {markdown && <Typography component="div" dangerouslySetInnerHTML={{ __html: markdown }} />}
+            {link && <Link href={link} />}
```

Comment:
Does the link auto-shows the href as it's visible text?

Reply:
no, it doesn't